microsoft/testsuites/performance/networkperf.py

```diff
@@ -424,6 +435,568 @@ def perf_sockperf_latency_tcp_synthetic_busy_poll(self, result: TestResult) -> N
             set_busy_poll=True,
         )
 
+    # Network device passthrough tests between host and guest
```

Comment:
If no code is reused, move it to a separated file for cloud_hypervisor, like `networkperf_ch.py`.

Reply:
@squirrelsc I will name it as `networkperf_passthrough.py` test. As of now we need run it for CH only. But this may extend for QEMU platform too and testcase scenarios remain same.

Reply:
It looks like test between host and guest, not other passthrough scenarios. How about networkperf_guest?

Reply:
@squirrelsc 

Guest VM will have passthrough interface. Host is physical server where we will run server daemon of required tool and client would run with Guest VM with passthrough NIC.